webapp/src/components/modals/styles.scss

```
@@ -0,0 +1,279 @@
+@keyframes fade-in {
+	from {
+		opacity: 0;
+	}
+	to {
+		opacity: 1;
+	}
+}
+
+@keyframes fade-out {
+	from {
+		opacity: 1;
+	}
+	to {
+		opacity: 0;
+	}
+}
+
+.custom-modal.modal {
+	opacity: 1;
+	padding-top: 230px;
+}
+
+.modal-open {
+	.modal {
+		overflow-x: hidden;
+		overflow-y: hidden;
+	}
+}
+
+.modal-dialog {
+	.modal-header {
+		display: flex;
+	}
+	.modal-body {
+		display: flex;
+		flex-direction: column;
+	}
+	width: 615px;
+	max-width: 615px;
+}
+
+.custom-modal-body {
+	height: 80%;
+	overflow-y: auto;
+}
+
+.fade-enter {
+	opacity: 0;
+	animation: fade-in 300ms forwards;
+}
+
+.fade-exit {
+	opacity: 1;
+	animation: fade-out 300ms forwards;
+}
+
+.btn-group {
+	display: flex;
+	justify-content: space-between;
+	.btn.btn-default {
+		background: none;
+		border-color: transparent;
+		border: transparent;
+		&:hover {
+			background: transparent;
+			color: #000000;
+		}
+		&:focus {
+			background: transparent;
+			color: #000000;
+		}
+	}
+}
+
+.radio.btn-group {
+	justify-content: flex-start;
+}
+
+.options {
+	.btn.btn-primary {
+		background: transparent;
+	}
+}
+
+.radio {
+	margin-top: 0;
+	margin-bottom: 0;
+}
+
+.radio-form {
+	margin-bottom: 0px;
```

```suggestion
	margin-top: 0;
	margin-bottom: 0;
}

.radio-form {
	margin-bottom: 0;
```
no need to add CSS unit for value `0`